Dockerfile

```diff
@@ -42,10 +42,16 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
         curl \
         ca-certificates \
+        dpkg-dev \
+        libjemalloc-dev \
         procps \
         && \
     rm -rf /var/lib/apt/lists/*
 
+RUN ln -s /usr/lib/$(dpkg-architecture -qDEB_HOST_MULTIARCH)/libjemalloc.so /usr/lib/libjemalloc.so
+
+ENV LD_PRELOAD=/usr/lib/libjemalloc.so
```

Comment:
I believe you can simply install `libjemalloc2` and do `LD_PRELOAD=libjemalloc.so.2`. Then you don't need `dpkg-dev` nor the manual symlink.

Reply:
directly installing libjemalloc2 still installed under `x86_64-linux-gnu` 

```bash
> sudo apt-get install libjemalloc2

❯ dpkg-query -L libjemalloc2
/.
/usr
/usr/lib
/usr/lib/x86_64-linux-gnu
/usr/lib/x86_64-linux-gnu/libjemalloc.so.2
/usr/share
/usr/share/doc
/usr/share/doc/libjemalloc2
/usr/share/doc/libjemalloc2/README
/usr/share/doc/libjemalloc2/changelog.Debian.gz
/usr/share/doc/libjemalloc2/copyright
```

Reply:
Yes I understand, but the path prefix is auto inferred based on the current platform so you just need the relative path.